server/service/IndividualServicesService.js

```diff
@@ -150,7 +182,18 @@ exports.disregardApplication = function (body, user, originator, xCorrelator, tr
   });
 }
 
-
+/*
+* Delete profile by name from application-data/application-data.json
+*/
+const deleteProfileByName = (applicationData, applicationNameToDelete) => {
```

Comment:
can we change the function name as it is not profile we are deleting

Reply:
done